deployment/registry.py

```diff
@@ -148,7 +148,10 @@ class ConflictResolution(Enum):
 def _select_conflict_resolution(
     registry_1_entry, registry_1_filepath, registry_2_entry, registry_2_filepath
```

Comment:
On the "green path" we'd deploy a contract, then replace it in the old registry, right?  This implies that the merge is one-way, so It may be useful to have an alternate strategy to "update" a base registry with a patch, instead of an interactive flow.  This can be achieved with some optional parameters here in your function to always select resolution 1 or 2.  You might even make this function private and then wrap it with differently parameterized calls.  Not a blocker, just commentary.

```python
def _perform_merge(...):
    ...
    
def interactive_merge(...):
    ...
    
def update_registry(...):
    ...
```

Reply:
Yeah that's a good point. I guess for some conflict resolution it won't be a 2nd filepath, just an updated contract to add to the registry at the end of an updated deployment eg. we update the Coordinator contract for Lynx.

We could handle that in a follow-up change.

Reply:
Filed #151 .